demos/browser/app/meetingV2/styleV2.scss

```diff
@@ -425,6 +425,22 @@ svg {
   }
 }
 
+.video-preview {
+  width:auto;
```

Comment:
Add a space after `:`. I understand that we don't run ESLint on demo files. Let's follow coding conventions from the existing code.

Reply:
done